ext/node/polyfills/testing.ts

```diff
@@ -147,30 +195,104 @@ function prepareDenoTest(name, options, fn, overrides) {
+  return function (t) {
+    const prevSuite = currentSuite;
+    const suite = currentSuite = new TestSuite(t);
+    try {
+      fn();
+    } finally {
+      currentSuite = prevSuite;
+    }
+    return SafePromisePrototypeFinally(SafePromiseAll(suite.steps), resolve);
+  };
+}
+
+function prepareDenoTestForSuite(name, options, fn, overrides) {
+  const prepared = prepareOptions(name, options, fn, overrides);
+
+  // deno-lint-ignore prefer-primordials
+  const { promise, resolve } = Promise.withResolvers();
+
+  const denoTestOptions = {
+    name: prepared.name,
+    fn: wrapSuiteFn(prepared.fn, resolve),
+    only: prepared.options.only,
+    ignore: prepared.options.todo || prepared.options.skip,
+    sanitizeExit: false,
+    sanitizeOps: false,
+    sanitizeResources: false,
+  };
+  Deno.test(denoTestOptions);
+  return promise;
+}
+
+export function test(name, options, fn, overrides) {
+  if (currentSuite) {
+    return currentSuite.addTest(name, options, fn, overrides);
+  }
+  return prepareDenoTest(name, options, fn, overrides);
 }
 
 test.skip = function skip(name, options, fn) {
-  return prepareDenoTest(name, options, fn, { skip: true });
+  return test(name, options, fn, { skip: true });
 };
 
 test.todo = function todo(name, options, fn) {
-  return prepareDenoTest(name, options, fn, { todo: true });
+  return test(name, options, fn, { todo: true });
 };
 
 test.only = function only(name, options, fn) {
-  return prepareDenoTest(name, options, fn, { only: true });
+  return test(name, options, fn, { only: true });
 };
 
-export function describe() {
-  notImplemented("test.describe");
+export function describe(name, options, fn) {
+  return suite(name, options, fn, {});
 }
 
-export function it() {
-  notImplemented("test.it");
+describe.skip = function skip(name, options, fn) {
+  return suite.skip(name, options, fn);
+};
+describe.todo = function todo(name, options, fn) {
+  return suite.todo(name, options, fn);
+};
+describe.only = function only(name, options, fn) {
+  return suite.only(name, options, fn);
+};
+
+export function suite(name, options, fn, overrides) {
+  if (currentSuite) {
+    return currentSuite.addSuite(name, options, fn, overrides);
+  }
+  return prepareDenoTestForSuite(name, options, fn, overrides);
 }
 
+suite.skip = function skip(name, options, fn) {
+  return suite(name, options, fn, { skip: true });
+};
+suite.todo = function todo(name, options, fn) {
+  return suite(name, options, fn, { todo: true });
+};
+suite.only = function only(name, options, fn) {
+  return suite(name, options, fn, { only: true });
+};
+
+export function it(name, options, fn) {
+  return test(name, options, fn, {});
+}
+
+it.skip = function skip(name, options, fn) {
+  return test.skip(name, options, fn);
+};
+
+it.todo = function todo(name, options, fn) {
+  return test.todo(name, options, fn);
+};
+
+it.only = function only(name, options, fn) {
+  return test.only(name, options, fn);
+};
+
```

Comment:
Can you add these lines?

```
test.test = test;
test.describe = describe;
test.it = it;
```
This will make this module more in-line with Node's implementation